client/src/pages/page.h

```diff
@@ -74,6 +74,7 @@ class Page : public QWidget
     void modelLoaded(DetailsType type);
     void modelItemChanged(const Akonadi::Item &item);
     void synchronizeCollection(const Akonadi::Collection &collection);
+    void synchronize();
```

Comment:
Can you call it syncRequired() too, like the signal in details? It didn't look like a signal otherwise, which confused me when reading the patch (I know, synchronizeCollection has that problem too, but let's not copy bad practices) ;)